renderer/components/test-results/ResultRow.js

```diff
@@ -172,30 +158,33 @@ class ResultRow extends React.Component {
   }
 
   renderIcon() {
-    const { name } = this.props
-
-    const group = testGroups[name]
-
-    return (
-      <Flex alignItems='center'>
-        <Box width={1/8}>
-          <ColorCode color={group.color} />
-        </Box>
-        <Box width={7/8}>
-          <Flex>
-            <Box pr={2}>
-              {React.cloneElement(
-                group.icon,
-                {size: 20, color: group.color}
-              )}
-            </Box>
-            <Box>
-              <Text color={group.color} bold>{group.name}</Text>
-            </Box>
-          </Flex>
-        </Box>
-      </Flex>
-    )
+    try {
+      const { name } = this.props
+      const group = testGroups[name]
+      return (
+        <Flex alignItems='center'>
+          <Box width={1/8}>
+            <ColorCode color={group.color} />
+          </Box>
+          <Box width={7/8}>
+            <Flex>
+              <Box pr={2}>
+                {React.cloneElement(
+                  group.icon,
+                  {size: 20, color: group.color}
+                )}
+              </Box>
+              <Box>
+                <Text color={group.color} bold>{group.name}</Text>
+              </Box>
+            </Flex>
+          </Box>
+        </Flex>
+      )
+    } catch (e) {
+      Sentry.captureException(new Error('name missing in result', e))
```

Comment:
Will this capture also the exception backtrace?

Maybe instead of creating a new Error, which may loose the exception stacktrace, we should log the error which you have caught in the catch and add a breadcrum with the selected testGroup name?

https://docs.sentry.io/enriching-error-data/breadcrumbs/?platform=electron#recording-crumbs